javascriptv3/example_code/README.rst

```diff
@@ -0,0 +1,43 @@
+###############################
+AWS SDK for JavaScript version 3 (v3) examples
+###############################
+
+These are examples for the AWS SDK for JavaScript (v3) public documentation.
+
+The prerelease version of the version 3 of the SDK is available `here <at https://github.com/aws/aws-sdk-js-v3>`_ .
```

Comment:
I would stay away from using "here" as a link target for accessibility reasons. Also, using different link text can help the user make sure they got to the right target.

Reply:
Changed to 
'The [preview version of the AWS SDK for JavaScript v3](https://github.com/aws/aws-sdk-js-v3) is available. '